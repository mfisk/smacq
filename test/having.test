#!/bin/sh

${BINDIR}smacqq -m -g <<EOF 
print srcip, count from pcapfile($TEST/0325@1112-snort.pcap) group by srcip having count() and count > 10
EOF
