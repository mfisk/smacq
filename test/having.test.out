192.168.0.54	107
